spec/integration/resource/api800/synergy/server_profile_template/update_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API500::Synergy::ServerProfileTemplate, integration: true, type: UPDATE do
+  let(:current_client) { $client_500_synergy }
```

Comment:
Replace 500 with 800

Reply:
@soodpr , Thanks for finding it. missed test filed files  for Synergy. Will update it.